backends/ceed-backend-weak.c

```diff
@@ -13,13 +12,13 @@ static int CeedInit_Weak(const char *resource, Ceed ceed) {
 
 // This function provides a debug target for weak symbols
 static int CeedRegister_Weak(const char *name, int num_prefixes, ...) {
-  if (getenv("CEED_DEBUG")) fprintf(stderr, "Weak %s\n", name);
-
   va_list prefixes;
   va_start(prefixes, num_prefixes);
   int ierr;
   for (int i=0; i<num_prefixes; i++) {
-    ierr = CeedRegister(va_arg(prefixes, const char*), CeedInit_Weak, CEED_MAX_BACKEND_PRIORITY);
+    const char *prefix = va_arg(prefixes, const char*);
+    CeedDebugEnv("** Weak Register **");
```

Comment:
This is okay, though we can't grep the results easily. If you skipped the `\n` in the Weak annotation, it would appear on one line.